go.mod

```diff
@@ -17,7 +17,6 @@ require (
 	github.com/ThalesIgnite/crypto11 v1.2.4
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d // indirect
 	github.com/cenkalti/backoff/v3 v3.0.0
-	github.com/evanphx/json-patch v4.11.0+incompatible
```

Comment:
`go mod tidy` removed this one - should I revert?

Reply:
that's ok